fp.md

```diff
@@ -14,7 +14,10 @@ people.forEach(function(person){
 });
 ```
 ```js
-// Your answer goes here
+
+people.map(function(person){
+  peoplesAges.push(person.age)
+});
```

Comment:
While this technically works (assuming `peopleAges` is defined), the beauty of `.map` is that it returns a new array whose contents are the return value of the code block.

```js
const peopleAges = people.map(function(person){
  return person.age
end
```
> Each iteration of `.map` will return an age. Each return value will represent a new item in the resulting array.